### DISCUSSION TITLE
Access control for event subscription

### DISCUSSION THREAD

**Discussion by devaskim** [eclipse-milo/milo#1333] [General]
Is there possible in Server SDK to allow/deny subscription to events (event types) for specific user?

For Variable Nodes I use custom attribute filter for UserAccessLevel attribute

**Comment by kevinherron** [comment]
I think you would have to intercept reads for the EventNotifier attribute on Objects and then set the SubscribeToEvents bit based on the user. To be honest I don't know if this will work correctly or not in the current implementation, you'll have to try it and see.
